src/client/windshaft.js

```diff
@@ -292,6 +294,8 @@ export default class Windshaft {
                     options: {
                         sql: aggSQL,
                         aggregation: agg,
+                        vector_extent: TILE_EXTENT,
+                        vector_simplify_extent: TILE_EXTENT,
```

Comment:
Not necessary (by default if you change `vector_extent`, `vector_simplify_extent` will be the same) , but I like that you added it here as it makes the intent clearer.